exams/pearson/download.py

```diff
@@ -93,14 +93,14 @@ def process(self):
                         if self.process_zip(local_path):
                             self.sftp.remove(remote_path)
                         log.debug("Processed remote file: %s", remote_path)
-                    except SSHException:
+                    except (EOFError, SSHException):
```

Comment:
nit: given that the exceptions are in a tuple, please put here (and any other instance below) a comma after the last exception: something like  `(EOFError, SSHException, )`